src/cli/cli_br.cpp

```diff
@@ -381,11 +379,28 @@ template <> otError Br::Process<Cmd("nat64prefix")>(Arg aArgs[])
  * @code
  * br prefixtable
  * prefix:fd00:1234:5678:0::/64, on-link:no, ms-since-rx:29526, lifetime:1800, route-prf:med,
- * router:ff02:0:0:0:0:0:0:1
+ * router:ff02:0:0:0:0:0:0:1 (M:0 O:0 Stub:1)
  * prefix:1200:abba:baba:0::/64, on-link:yes, ms-since-rx:29527, lifetime:1800, preferred:1800,
- * router:ff02:0:0:0:0:0:0:1
+ * router:ff02:0:0:0:0:0:0:1 (M:0 O:0 Stub:1)
  * Done
  * @endcode
+ * @par
+ * Get the discovered prefixes by Border Routing Manager on the infrastructure link.
+ *
```

Comment:
Blank lines will break the reference import.  Can you put a `@par` here please

Reply:
Removed the new line in new push.